sso-oidc/elasticsearch-with-oidc.yml

```diff
@@ -0,0 +1,139 @@
+#
+#http.port: 9200
+#
+# For more information, consult the network module documentation.
+#
+# --------------------------------- Discovery ----------------------------------
+#
+# Pass an initial list of hosts to perform discovery when this node is started:
+# The default list of hosts is ["127.0.0.1", "[::1]"]
+#
+#discovery.seed_hosts: ["host1", "host2"]
+#
+# Bootstrap the cluster using an initial set of master-eligible nodes:
+#
+#cluster.initial_master_nodes: ["node-1", "node-2"]
+#
+# For more information, consult the discovery and cluster formation module documentation.
+#
+# ---------------------------------- Various -----------------------------------
+#
+# Allow wildcard deletion of indices:
+#
+#action.destructive_requires_name: false
+
+#----------------------- BEGIN SECURITY AUTO CONFIGURATION -----------------------
+#
+# The following settings, TLS certificates, and keys have been automatically      
+# generated to configure Elasticsearch security features on 05-04-2022 13:52:15
+#
+# --------------------------------------------------------------------------------
+
+# Enable security features
+xpack.security.enabled: true
+
+xpack.security.enrollment.enabled: true
+
+# Enable encryption for HTTP API client connections, such as Kibana, Logstash, and Agents
+xpack.security.http.ssl:
+  enabled: true
+  keystore.path: certs/http.p12
+
+# Enable encryption and mutual authentication between cluster nodes
+xpack.security.transport.ssl:
+  enabled: true
+  verification_mode: certificate
+  keystore.path: certs/transport.p12
+  truststore.path: certs/transport.p12
+
+xpack.security.authc.token.enabled: true
+xpack.security.authc.realms.oidc.oidc1:
+  order: 1
+  rp.client_id: "<<CLIENT ID GOES HERE>>"
+  rp.response_type: code
+  rp.redirect_uri: "http://localhost:5601/api/security/oidc/callback"
+  op.issuer: "https://localhost:9443/oauth2/token"
+  op.authorization_endpoint: "https://localhost:9443/oauth2/authorize"
+  op.token_endpoint: "https://localhost:9443/oauth2/token"
+  op.jwkset_path: "https://localhost:9443/oauth2/jwks"
+  op.endsession_endpoint: "https://localhost:9443/oidc/logout"
+  op.userinfo_endpoint: "https://localhost:9443/oauth2/userinfo"
+  rp.post_logout_redirect_uri: "http://localhost:5601/security/logged_out"
+  claims.principal: sub
+  ssl.verification_mode: none
+  claims.groups: groups
+  claims.name: name
+  claims.mail: email
+
+# Create a new cluster with the current node only
+# Additional nodes can still join the cluster later
+cluster.initial_master_nodes: ["Avarjana-Personal"]
+
+# Allow HTTP API connections from localhost and local networks
+# Connections are encrypted and require user authentication
+http.host: [_local_, _site_]
+
+# Allow other nodes to join the cluster from localhost and local networks
+# Connections are encrypted and mutually authenticated
+#transport.host: [_local_, _site_]
+
+#----------------------- END SECURITY AUTO CONFIGURATION -------------------------
```

Comment:
shall we add a new line

Reply:
Fixed with https://github.com/wso2-extensions/identity-elk-integration/pull/14